processing/src/main/java/org/apache/carbondata/processing/store/CarbonFactDataHandlerColumnar.java

```diff
@@ -239,6 +239,7 @@ public void addDataToStore(CarbonRow row) throws CarbonDataWriterException {
    * @return false if any varchar column page cannot add one more value(2MB)
    */
   private boolean isVarcharColumnFull(CarbonRow row) {
+    //TODO: test and remove this as now  UnsafeSortDataRows can exceed 2MB
```

Comment:
This is for a column page cannot exceed 2GB （actually it is 1.67GB since snappy cannot compress a bigger size in one run）, so there is no need to add a comment here

Reply:
complex column also can grow very big (so checking only for var char is not good)
Also now columns grow more than 2MB. so, we need to modify this check.
This can be handled in separate PR.

now no impact from this method as "if 2MB itself space not there, more than 2MB space will never be there". so functionality remains same.